.ci/docker/common/install_user.sh

```diff
@@ -4,8 +4,13 @@ set -ex
 
 # Mirror jenkins user in container
 # jenkins user as ec2-user should have the same user-id
-echo "jenkins:x:1000:1000::/var/lib/jenkins:" >> /etc/passwd
-echo "jenkins:x:1000:" >> /etc/group
+if [ "$IS_ARC" = "true" ]; then
+  echo "jenkins:x:1001:1001::/var/lib/jenkins:" >> /etc/passwd
+  echo "jenkins:x:1001:" >> /etc/group
```

Comment:
Seems like there's some history I'm missing here that I'd like to understand. I'd like to understand why we need a Jenkins user if we're not using Jenkins?

Reply:
One thing that bothers me, we're suppose to be running DID rootless. 

Then we create a jenkins user, set it to the same UID as the runner user, and add it to /etc/sudoers. So we're not doing rootless then...?

Reply:
@jeanschmidt that's correct. If we are adding the runner user to sudoer's and allowing that user to switch to root then we indeed are not doing dind-rootless. We'd need to lockdown the runner user to prevent it from calling sudo if we want to be fully rootless.